velox/exec/Operator.h

```diff
@@ -454,7 +454,9 @@ class Operator : public BaseRuntimeStatWriter {
   }
 
   /// Returns a list of identity projections, e.g. columns that are projected
-  /// as-is possibly after applying a filter.
+  /// as-is possibly after applying a filter. It is used to allow dynamic
+  /// filters pushdown to go through. More examples of identity projections: all
```

Comment:
nit: "More examples" ->"Examples" since this is the first mentioning of examples.

Reply:
Done.